src/BCMath.cxx

```diff
@@ -72,8 +72,8 @@ double BCMath::LogPoisson(double x, double lambda)
 // ---------------------------------------------------------
 double BCMath::LogApproxBinomial(unsigned n, unsigned k, double p)
 {
-    if (k < n)										// impose k < n requirement
-        return LogApproxBinomial(k, n, p);
+    if (k > n)										// impose k < n requirement
+        return -std::numeric_limits<double>::infinity();
```

Comment:
Perhaps. At some point, someone coded in return values of 1e-99, or something similar. I replaced these all with -inf. (They are in calculation of log likelihoods.) I tried to put quiet_NaN() wherever it made sense; I'm sure I missed cases.